servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/SessionTests.java

```diff
@@ -105,7 +105,10 @@ public void tearDown() throws ServiceBusException, InterruptedException, Executi
 	
     @AfterClass
     public static void cleanupAfterAllTest() throws ExecutionException, InterruptedException, IOException {
-        if(SessionTests.entityNameCreatedForAllTests != null)
+        if (managementClient == null) {
+            return;
+        }
+	    if(SessionTests.entityNameCreatedForAllTests != null)
         {
```

Comment:
Eventually you'll enable spotbugs / checkstyle on here as well? I see braces on new lines, etc that can be fixed eventually.

Reply:
updated. 

Yes. The enabled spotbugs and checkstyle PR is closed but not merged. Because of the change for enabling the spotbugs and checkstyle is less line. The changes moved to here.